From: Michael Biebl <biebl@debian.org>
Date: Wed, 6 May 2015 18:17:51 +0200
Subject: Fix iscsiadm path

The open-scsi package in Debian installs the iscisadm binary as
/usr/bin/iscsiadm.
---
 src/settings/plugins/ibft/plugin.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/settings/plugins/ibft/plugin.c b/src/settings/plugins/ibft/plugin.c
index 6f9b8c8..ce89599 100644
--- a/src/settings/plugins/ibft/plugin.c
+++ b/src/settings/plugins/ibft/plugin.c
@@ -60,7 +60,7 @@ read_connections (SettingsPluginIbft *self)
 	GError *error = NULL;
 	NMIbftConnection *connection;
 
-	if (!read_ibft_blocks ("/sbin/iscsiadm", &blocks, &error)) {
+	if (!read_ibft_blocks ("/usr/bin/iscsiadm", &blocks, &error)) {
 		nm_log_dbg (LOGD_SETTINGS, "ibft: failed to read iscsiadm records: %s", error->message);
 		g_error_free (error);
 		return;
